README.md

```diff
@@ -82,20 +82,20 @@ paymentForm.addEventListener("submit", payWithPaystack, false);
 function payWithPaystack(e) {
   e.preventDefault();
 
-  let handler = PaystackPop.setup({
-    key: 'pk_test_xxxxxxxxxx',
-    email: document.getElementById("email-address").value,
-    amount: document.getElementById("amount").value * 100,
-    ref: paystackTxRef // Reference returned from plugin
-    onClose: function(){
-      alert('Window closed.');
-    },
-    callback: function(response){
-      // Call Medusa checkout complete here
-    }
-  });
-
-  handler.openIframe();
+  const paystack = new PaystackPop();
+
+  paystack.newTransaction({
+	key: 'pk_test_xxxxxxxxxx', // Your Paystack public key
+	email: document.getElementById("email-address").value,
+	amount: document.getElementById("amount").value * 100,
```

Comment:
The `*100` here was what tripped you up in #74 yes? I think we should remove it. The value used here should be coming from the Medusa Cart anyways

Reply:
Yes haha! Will update this.

Reply:
Just saw you took care of it, thanks!

All good now I hope?

Reply:
@mannyAndem no those changes are in the [different branch](https://github.com/mannyAndem/medusa-payment-paystack/compare/update-docs...a11rew%3Amedusa-payment-paystack%3Aupdate-docs) I used to validate the changes locally. You'd have to make the changes on this branch to update the PR

Reply:
Ohh! Really new to this, will take care of it now.